src/enforester.js

```diff
@@ -63,7 +63,7 @@ const EXPR_LOOP_OPERATOR = {};
 const EXPR_LOOP_NO_CHANGE = {};
 const EXPR_LOOP_EXPANSION = {};
 
-function getLineNumber(x: Syntax | T.Term) {
+function getLineNumber(x: any) {
```

Comment:
Sounds good. You might want to add `// @flow` to the top of all files using type annotations at the same time. Unless you want to save that for another PR.